panel/pane/base.py

```diff
@@ -5,17 +5,14 @@
 from __future__ import annotations
 
 import warnings
-
 from functools import partial
-from typing import (
-    TYPE_CHECKING, Any, Callable, List, Optional, TypeVar, Type
-)
+from typing import (TYPE_CHECKING, Any, Callable, List, Optional, Type,
+                    TypeVar, Union)
 
 import param
-
-from bokeh.models.layouts import (
-    GridBox as _BkGridBox, Tabs as _BkTabs, Panel as _BkPanel
-)
+from bokeh.models.layouts import GridBox as _BkGridBox
+from bokeh.models.layouts import Panel as _BkPanel
+from bokeh.models.layouts import Tabs as _BkTabs
```

Comment:
I know isort does this but I really don't like it. Is there an isort configuration that avoids this because at that point I'd be happy to enable it?

Reply:
See https://github.com/bokeh/bokeh/blob/530c735f75f065b33acced0096f58e265aae4a65/bokeh/.isort.cfg#L5.